Dockerfile

```diff
@@ -258,9 +258,10 @@ RUN go env GOVERSION > /GOVERSION
 FROM base AS test-integration
 ARG DEBIAN_FRONTEND=noninteractive
 # `expect` package contains `unbuffer(1)`, which is used for emulating TTY for testing
+# libsystemd-dev: for sdjournal
 RUN apt-get update && \
   apt-get install -qq -y \
-  expect git
+  expect git libsystemd-dev
```

Comment:
Not a huge fan of cgo.

If cgo is necessary we should split this plugin into a third party repo and call it via `--log-opt binary://`

Reply:
And what's the license of this?

Reply:
I think we should not think about cgo, I think the dynamic link library may cause some potential problems in some distribution.

Reply:
fixed